src/emulator/remoteconfig/server.ts

```diff
@@ -0,0 +1,79 @@
+import * as express from "express";
+import { RemoteConfigEmulator } from "./index";
+import * as cors from "cors";
+
+/**
+ * @param defaultProjectId Default project ID used to start the emulator.
+ * @param emulator Remote Config emulator.
+ * @returns Express app.
+ */
+export function createApp(
+  defaultProjectId: string,
+  emulator: RemoteConfigEmulator
+): Promise<express.Express> {
+  const app = express();
+  app.use(cors("*" as any));
+
+  app.put("/revert", (req, res) => {
+    emulator.loadTemplate();
+    res.json(emulator.template);
+  });
+
+  // Please note you should set the Remote Config minimal fetch interval to 0, so it refreshes every time
+  // Otherwise caching will cause you headaches
+  app.post("/v1/projects/:projectId/namespaces/firebase:fetch", (req, res) => {
```

Comment:
This is gonna see `:fetch` as an express param, so this will match `/v1/projects/:projectId/namespaces/firebase:foo `as well.

I _think_ you can escape it like `/v1/projects/:projectId/namespaces/firebase[:]fetch` to prevent this

Reply:
done